apps/sasquatch/src/main/java/com/microsoft/appcenter/sasquatch/activities/MainActivity.java

```diff
@@ -69,6 +71,14 @@
 
     static final String MAX_STORAGE_SIZE_KEY = "maxStorageSize";
 
+    public static final String MSA_USER_ID_KEY = "MSAUserId";
```

Comment:
Let's change that to TICKET_KEY instead of USER_ID, we are not using setUserId for that.